File: backend/ExpressJS/Models/Wish.js
const mongoose = require('mongoose');

const wishSchema = new mongoose.Schema({
    description: { type: String, required: true },
    quantite: { type: String, required: true },
    categorie_wish: { required: true } = Object.freeze({ "Toy": "Toy", "Book ": "Book", })
});

module.exports = mongoose.model('Wish', WishSchema);

File: mobile/Flutter/lib/Models/wish.dart
class Wish  {

  String? _id;
  String? _description;
  String? _categorieWish;

  Wish(this._id, this._description, this._categorieWish);

  String? get categorieWish => _categorieWish;

  String? get id => _id;

  set id(String? value) {
    _id = value;
  }

  set categorieWish(String? value) {
    _categorieWish = value;
  }

  String? get description => _description;

  set description(String? value) {
    _description = value;
  }

}